blog/2020-05-25-infinispan-11.0.0.CR1.adoc

```diff
@@ -0,0 +1,92 @@
+---
+layout: blog
+title: Infinispan 11.0.0.CR1
+date: '2020-05-25T12:00:00.000-00:00'
+author: Tristan Tarrant
+tags: [ "release" ]
+---
+
+= Infinispan 11.0.0.CR1
+
+Dear Infinispan community,
+
+we've just released 11.0.0.CR1 which comes loaded with quite a few features and improvements.
+
+== Cross-Site Replication Improvements
```

Comment:
this seems a lot of text comparing with the other sections. 
What about only add the "sub-titles"
```
* Support for multiple site masters
* Conflict detection and resolution for Asynchronous Cross-Site Replication
```

Reply:
It is the highlighted feature. I'll leave it like that.